.travis.yml

```diff
@@ -4,15 +4,15 @@ os:
   - osx
 julia:
   - 1.0
-  - 1.3
+  - 1.4
   - nightly
 codecov: true
 jobs:
   allow_failures:
     - julia: nightly
   include:
     - stage: "Documentation"
-      julia: 1.3
+      julia: 1.4
```

Comment:
I added an examples stage to this script, so a rebase with master will be necessary